src/server/helpers/source-api-request.ts

```diff
@@ -195,8 +195,8 @@ export async function requestData<T>(
     const shouldCaptureMessage =
       error.isAxiosError || (!(error instanceof Error) && !!error?.message);
 
-    const api = Object.entries(ApiUrls).find(
-      ([, url]) => requestConfig.url === url
+    const api = Object.entries(ApiUrls).find(([, url]) =>
+      requestConfig.url?.startsWith(url)
```

Comment:
Sommige URL's worden naderhand uitgebreid met b.v. get parameters. Dat zorgt ervoor dat de .find functie met strikte vergelijking soms geen resultaat oplevert. Daardoor krijgen we soms in Sentry niet duidelijk door welke API call fout gaat.